apps/web/src/components/editor/tiptap.tsx

```diff
@@ -165,7 +165,15 @@ function TipTap(props: TipTapProps) {
   const tiptapOptions = useMemo<Partial<TiptapOptions>>(() => {
     return {
       editorProps: {
-        handleKeyDown(view, event) {
+        handleKeyDown(_, event) {
+          if (event.ctrlKey && event.key === "ArrowRight") {
+            event.preventDefault();
+            useEditorStore.getState().openNextSession("right");
+          }
+          if (event.ctrlKey && event.key === "ArrowLeft") {
+            event.preventDefault();
+            useEditorStore.getState().openNextSession("left");
+          }
```

Comment:
Not sure this is the right place to handle this. Also Ctrl + Arrow Keys are used for caret navigation so we can't use those.

Reply:
Nice catch on the Ctrl + Arrow shortcut. I've moved this shortcut to Ctrl + Alt + Arrow, let me know if this is okayy